app/src/main/java/com/example/pricingpal/model/modules/SupabaseModule.kt

```diff
@@ -7,8 +7,10 @@ import dagger.hilt.InstallIn
 import dagger.hilt.components.SingletonComponent
 import io.github.jan.supabase.SupabaseClient
 import io.github.jan.supabase.createSupabaseClient
-import io.github.jan.supabase.gotrue.GoTrue
-import io.github.jan.supabase.gotrue.gotrue
+import io.github.jan.supabase.gotrue.Auth
+//import io.github.jan.supabase.gotrue.GoTrue
```

Comment:
Did mean to keep line 11 and 13?

Reply:
No, I didn't. Thanks for catching that. I removed it!